docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,739 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    pos = np.argmax(p)
</pre><p>You might as well just call this <code>mp_discord_idx = np.argmax(P)</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
will be fixed in next commits.